src/pivot-table/data/extract-left.ts

```diff
@@ -19,6 +20,7 @@ const extractLeftGrid = (
     root: Cell | null,
     parent: Cell | null,
     nodes: EngineAPI.INxPivotDimensionCell[],
+    pgInfo: PageInfo,
```

Comment:
You don't need to pass pageInfo in as a param here. It's already available in the function scope